frontend/libs/studio-components/src/components/StudioTextField/StudioTextfieldSchema/StudioTextfieldSchema.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useMemo, useState } from 'react';
+import {
+  StudioToggableTextfield,
+  type StudioToggableTextfieldProps,
+} from '../StudioToggableTextfield';
+import type { JsonSchema } from '../../../../../../packages/shared/src/types/JsonSchema';
+import { useTranslation } from 'react-i18next';
+import { StudioJSONValidatorUtils } from '../StudioJSONValidatorUtils';
+
+export type StudioTextfieldSchemaProps = {
+  schema: JsonSchema;
+  propertyPath: string;
+  jsonValidator: any;
+} & StudioToggableTextfieldProps;
+
+export const StudioTextfieldSchema = ({
+  jsonValidator,
+  schema,
+  inputProps,
+  propertyPath,
+  ...rest
+}: StudioTextfieldSchemaProps) => {
+  const [errorMessage, setErrorMessage] = useState<string | null>();
+  const studioJSONValidator = new StudioJSONValidatorUtils(jsonValidator);
+  const { t } = useTranslation();
+
+  const propertyId = useMemo(
+    () => (schema && propertyPath ? `${schema.$id}#/${propertyPath}` : null),
+    [schema, propertyPath],
+  );
+
```

Comment:
I actually believe we can simplify this to the following:

```suggestion
  const propertyId = (schema && propertyPath) ? `${schema.$id}#/${propertyPath}` : null;
```

Reply:
loved it❤️